src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -530,6 +523,24 @@ private void getCanvasResponse(HttpServletRequest request,
 		out.flush();
 	}
 
+	public String unmaskUrl(String url, TcSessionData tc,
+			String stringToReplaceUser, String stringToReplaceCourse) {
+		if (tc != null){
+			String uniqname = (String) tc.getCustomValuesMap().get(CUSTOM_CANVAS_USER_LOGIN_ID);
+			String courseId = (String) tc.getCustomValuesMap().get(CUSTOM_CANVAS_COURSE_ID);
+			M_log.debug("uniqname: " + uniqname);
+			String replaceValue = "as_user_id=sis_login_id:" + uniqname;
```

Comment:
Better variable name please.  Since multiple things are replaced 'replaceValue' is ambiguous.